en/docs/chapter_searching/replace_linear_by_hashing.md

```diff
@@ -42,6 +42,6 @@ The implementation code is shown below, requiring only a single loop:
 [file]{two_sum}-[class]{}-[func]{two_sum_hash_table}
 ```
 
-This method reduces the time complexity from $O(n^2)$ to $O(n)$ by using hash search, greatly improving the running efficiency.
+This method reduces the time complexity from $O(n^2)$ to $O(n)$ by using hash search, significantly improving the running efficiency.
```

Comment:
improving the running efficiency => enhancing runtime efficiency

Reply:
fixed